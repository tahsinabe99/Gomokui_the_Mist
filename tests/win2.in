place C1
place D1
place C2
place D2
place C3
place D3
place C4
place D4
place C5
