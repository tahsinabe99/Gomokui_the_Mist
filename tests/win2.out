Black wins!
C1D1C2D2C3D3C4D4C5
Thank you for playing!
